typespec-tests/src/main/java/com/cadl/naming/models/Data.java

```diff
@@ -13,12 +13,13 @@
 import java.io.IOException;
 
 /**
- * The Data model.
+ * Dummy doc to make the javadoc break at the 'at' symbol. The type of the Data depends on
+ * &#064;data.kind.letusmakeitlongsoitwouldbreakbeforethis field.
```

Comment:
Target is actually only here.

javadoc would fail if this line starts with `@data.kind`. It confuses with Javadoc tag.